### PR TITLE
Updating documentation for Channel Summary property

### PR DESCRIPTION
**Pull request by kallu800**

### CODE REVIEW

File: api-reference/beta/api/channel-get.md [hunk 1]
```diff
@@ -192,6 +192,23 @@ Content-type: application/json
 }
 ```
 
+The following example shows the response for a standard channel with the **summary** property.
```
Comment (Lauragra) [inline api-reference/beta/api/channel-get.md:195]: Is the name of the property **summary** or **channelSummary**? I suggest adding a second example that shows the `select` parameter with the channelSummary property in the request and then include the response. Also, please add the requirement to use the `select` parameter to get the channel summary to the Optional query parameters section.

File: changelog/Microsoft.Teams.GraphSvc.json [hunk 2]
```diff
@@ -733,6 +733,32 @@
...
+          "ChangeType": "Addition",
+          "Description": "Added the [channelSummary](https://learn.microsoft.com/en-us/graph/api/resources/channelSummary?view=graph-rest-beta) resource type.",
+          "Target": "channelSummary"
+        },
+        {
+          "Id": "8A36C99E-BF27-45D1-9888-F55420EEB5C2",
+          "ApiChange": "Property",
+          "ChangedApiName": "summary",
+          "ChangeType": "Addition",
+          "Description": "Added the **summary** property to [channel](https://learn.microsoft.com/en-us/graph/api/resources/channel?view=graph-rest-beta) resource.",
```
Comment (Lauragra) [inline changelog/Microsoft.Teams.GraphSvc.json:752]: We should change this to "channelSummary", right?

File: api-reference/beta/resources/channel.md [hunk 3]
```diff
@@ -65,6 +65,7 @@ where files are shared, and where tabs are added.
 |email|String| The email address for sending messages to the channel. Read-only.|
 |webUrl|String|A hyperlink that will go to the channel in Microsoft Teams. This is the URL that you get when you right-click a channel in Microsoft Teams and select Get link to channel. This URL should be treated as an opaque blob, and not parsed. Read-only.|
 |membershipType|[channelMembershipType](../resources/channel.md#channelmembershiptype-values)|The type of the channel. Can be set during creation and can't be changed. The possible values are: `standard`, `private`, `unknownFutureValue`, `shared`. The default value is `standard`. Note that you must use the `Prefer: include-unknown-enum-members` request header to get the following value in this [evolvable enum](/graph/best-practices-concept#handling-future-members-in-evolvable-enumerations): `shared`.|
+|channelSummary|[channelSummary](../resources/channelSummary.md)|Contains summary information about the channel, including number of guests, members, and owners, and whether the channel includes members from other tenants. This property is only rendered if it is specified in the `$select` clause of the Get channel API.|
```
Comment (Lauragra) [inline api-reference/beta/resources/channel.md:68]: According to the api.md file, the name of the property is **summary**; please update here. Note that we list properties in the Properties table in alphabetical order so please include it in the right order here.

File: api-reference/beta/includes/snippets/csharp/get-channel-summary-csharp-snippets.md [hunk 4]
```diff
@@ -0,0 +1,18 @@
+---
+description: "Automatically generated file. DO NOT MODIFY"
```
Comment (Lauragra) [inline api-reference/beta/includes/snippets/csharp/get-channel-summary-csharp-snippets.md:2]: Global: I'm not sure why you're adding snippet files. FYI, SDK snippets are generated via our weekly snippet automation process, we don't need to add them individually to PRs. More information here: https://dev.azure.com/msazure/One/_wiki/wikis/Microsoft%20Graph%20Partners/211752/Generate-code-snippets

File: api-reference/beta/api/channel-get.md [hunk 5]
```diff
@@ -41,6 +41,8 @@ GET /teams/{team-id}/channels/{channel-id}
 
 This method supports the `$filter`, `$select`, and `$expand` [OData query parameters](/graph/query-parameters) to help customize the response.
 
+>**Note:** The channel summary property can only be retrieved via the $select clause as can be seen in examples below.
```
Comment (Lauragra) [inline api-reference/beta/api/channel-get.md:44]: Please revise this note as follows:

The **summary** property can only be retrieved via the `select` parameter, as shown in Example 2 in this topic.

File: api-reference/beta/api/channel-get.md [hunk 6]
```diff
@@ -192,6 +194,63 @@ Content-type: application/json
 }
 ```
 
+### Request
```
Comment (Lauragra) [inline api-reference/beta/api/channel-get.md:197]: When we have multiple examples in a topic, we create separate H3 sections for each example. Please change the H2 here from "Example" to "Examples", then add each example under new H3s, titled like this:

### Example 1: {title}
### Example 2: {title}

The Request and Response headings will then be H4s.

File: api-reference/beta/api/channel-get.md [hunk 7]
```diff
@@ -99,9 +103,9 @@ GET https://graph.microsoft.com/beta/teams/893075dd-2487-4122-925f-022c42e20265/
 ---
 
 
-### Response
+#### Response 1
```
Comment (Lauragra) [inline api-reference/beta/api/channel-get.md:106]: Please don't number the Request and Response headings, just the Example H3s.

File: api-reference/beta/api/channel-get.md [hunk 8]
```diff
@@ -192,6 +202,65 @@ Content-type: application/json
 }
 ```
 
+### Example 2: Get a Channel Summary property
```
Comment (Lauragra) [inline api-reference/beta/api/channel-get.md:205]: Change Channel Summary to channelSummary.

File: api-reference/beta/api/channel-get.md [hunk 9]
```diff
@@ -192,6 +202,65 @@ Content-type: application/json
...
+
+#### Request
+
+Here is an example of the request to get the Channel Summary property.
```
Comment (Lauragra) [inline api-reference/beta/api/channel-get.md:209]: Revise as: The following example shows a request to get the channelSummary property.

File: api-reference/beta/api/channel-get.md [hunk 10]
```diff
@@ -192,6 +202,65 @@ Content-type: application/json
...
+
+
+# [HTTP](#tab/http)
```
Comment (Lauragra) [inline api-reference/beta/api/channel-get.md:212]: Please remove the HTTP tab.

File: api-reference/beta/api/channel-get.md (hunk 8 above)
Comment (sumitgupta3) [reply api-reference/beta/api/channel-get.md]: made changes as requested

File: api-reference/beta/api/channel-get.md [hunk 11]
```diff
@@ -192,6 +202,65 @@ Content-type: application/json
...
+```msgraph-interactive
```
[the remark this answers is not in the document]
Comment (sumitgupta3) [reply api-reference/beta/api/channel-get.md:211]: I have added the change, hope this is what you meant.

File: api-reference/beta/resources/channel.md [hunk 12]
```diff
@@ -67,6 +67,7 @@ where files are shared, and where tabs are added.
 |membershipType|[channelMembershipType](../resources/channel.md#channelmembershiptype-values)|The type of the channel. Can be set during creation and can't be changed. The possible values are: `standard`, `private`, `unknownFutureValue`, `shared`. The default value is `standard`. Note that you must use the `Prefer: include-unknown-enum-members` request header to get the following value in this [evolvable enum](/graph/best-practices-concept#handling-future-members-in-evolvable-enumerations): `shared`.|
 |createdDateTime|dateTimeOffset|Read only. Timestamp at which the channel was created.|
 |moderationSettings|[channelModerationSettings](../resources/channelmoderationsettings.md)|Settings to configure channel moderation to control who can start new posts and reply to posts in that channel.|
+|summary|[summary](../resources/channelsummary.md)|Contains summary information about the channel, including number of guests, members, owners, and an indicator for members from other tenants. The **summary** property will only be returned if it is specified in the `$select` clause of the [Get channel](../api/channel-get.md) method.|
```
[the remark this answers is not in the document]
Comment (sumitgupta3) [reply api-reference/beta/resources/channel.md:70]: Yes Datamember name is summary. Its channel's summary denoted by variable "summary"
image

[the remark this answers is not in the document]
Comment (sumitgupta3) [reply api-reference/beta/resources/channel.md:70]: but as you can see we a have resource in file downward of this which tells this is resource with name channelSummary. Yes, in response its returned with name 'summary'.   here: api-reference/beta/resources/channelsummary.md 

I am not clear what change I should do to this as it looks appropriate.

File: changelog/Microsoft.Teams.GraphSvc.json [hunk 13]
```diff
@@ -699,31 +699,31 @@
...
+          "Description": "Added the [markChatReadForUser](https://learn.microsoft.com/en-us/graph/api/chat-markchatreadforuser?view=graph-rest-1.0) method to the [chat](https://learn.microsoft.com/en-us/graph/api/chat?view=graph-rest-1.0) resource.",
           "Target": "chat"
         },
         {
           "Id": "853fa7eb-f4e4-43ba-b7c9-80bbb930bcef",
           "ApiChange": "Method",
           "ChangedApiName": "markChatUnreadForUser",
           "ChangeType": "Addition",
-          "Description": "Added the [markChatUnreadForUser](https://learn.microsoft.com/en-us/graph/api/chat?view=graph-rest-1.0) method to the [chat](https://learn.microsoft.com/en-us/graph/api/chat?view=graph-rest-1.0) resource.",
+          "Description": "Added the [markChatUnreadForUser](https://learn.microsoft.com/en-us/graph/api/chat-markchatunreadforuser?view=graph-rest-1.0) method to the [chat](https://learn.microsoft.com/en-us/graph/api/chat?view=graph-rest-1.0) resource.",
```
Comment (devjha-ms) [inline changelog/Microsoft.Teams.GraphSvc.json:718]: Please remove these additional changes which might have come due to conflict resolution.

File: api-reference/beta/api/channel-get.md [hunk 14]
```diff
@@ -192,6 +195,36 @@ Content-type: application/json
 }
 ```
 
+### Example 2: Get a channelSummary property
+
+#### Request
+
+The following example shows a request to get the **channelSummary** property.
+
+```http
```
Comment (Lauragra) [inline api-reference/beta/api/channel-get.md:204]: Please add the HTML comments to the request and response examples for API Doctor validation.

File: api-reference/beta/resources/channel.md (hunk 12 above)
[the remark this answers is not in the document]
Comment (Lauragra) [reply api-reference/beta/resources/channel.md:70]: In the Type column, please change the link name from summary to channelSummary.

File: changelog/Microsoft.Teams.GraphSvc.json (hunk 13 above)
Comment (sumitgupta3) [reply changelog/Microsoft.Teams.GraphSvc.json:718]: This fixes the current docs and are fine to have it.

File: api-reference/beta/resources/channel.md (hunk 12 above)
[the remark this answers is not in the document]
Comment (sumitgupta3) [reply api-reference/beta/resources/channel.md:70]: added

File: api-reference/beta/api/channel-get.md (hunk 14 above)
Comment (sumitgupta3) [reply api-reference/beta/api/channel-get.md:225]: added, hope its the way we need, pls check.

File: changelog/Microsoft.Teams.GraphSvc.json (hunk 13 above)
Comment (devjha-ms) [reply changelog/Microsoft.Teams.GraphSvc.json:718]: Is there any bug to track this doc changes?

**Review by Lauragra — APPROVED** [review]

**Pull request merged**